spec/controllers/frontend_log_controller_spec.rb

```diff
@@ -214,18 +188,35 @@
         expect(Analytics).to receive(:new).with(hash_including(user: user))
       end
 
-      it 'succeeds' do
-        expect(fake_analytics).to receive(:track_event).with("Frontend: #{event}", payload)
+      context 'with invalid event name' do
+        it 'does not log' do
+          expect(fake_analytics).to_not receive(:track_event)
 
-        action
+          action
 
-        expect(response).to have_http_status(:ok)
-        expect(json[:success]).to eq(true)
+          expect(response).to have_http_status(:bad_request)
+          expect(json[:success]).to eq(false)
```

Comment:
Minor: I might suggest asserting for the extra property as well (`error_message`), if we care enough to include it.

Reply:
yep, totally. added in d099da2181